hawkbit-rest/hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/DdiDeploymentBaseTest.java

```diff
@@ -722,11 +719,15 @@ public void missingFinishedAttributeInFeedbackReturnsBadRequest() throws Excepti
 
         final Action action = deploymentManagement.findActionsByTarget(target.getControllerId(), PAGE).getContent()
                 .get(0);
-        final String missingFinishedResultInFeedback = JsonBuilder
-                .missingFinishedResultInFeedback(action.getId().toString(), "closed", "test");
-        postDeploymentFeedback("1080", action.getId(), missingFinishedResultInFeedback,
-                status().isBadRequest()).andExpect(jsonPath("$.*", hasSize(3))).andExpect(
-                        jsonPath("$.exceptionClass", equalTo(MessageNotReadableException.class.getCanonicalName())));
+
+        final DdiStatus ddiStatus = new DdiStatus(DdiStatus.ExecutionStatus.CLOSED, new DdiResult(null, null),
```

Comment:
can't we somehow reuse the getJsonActionFeedback here by making it more generic?

Reply:
Now this method also uses `getJsonActionFeedback`